gui/autodump.lua

```diff
@@ -308,12 +308,25 @@ end
 
 function Autodump:do_dump(pos)
     pos = pos or dfhack.gui.getMousePos()
-    if not pos then return end
-    local tileattrs = df.tiletype.attrs[dfhack.maps.getTileType(pos)]
-    local basic_shape = df.tiletype_shape.attrs[tileattrs.shape].basic_shape
-    local on_ground = basic_shape == df.tiletype_shape_basic.Floor or
-            basic_shape == df.tiletype_shape_basic.Stair or
-            basic_shape == df.tiletype_shape_basic.Ramp
+    if not pos then --We check this before calling
+        dfhack.printerr('Autodump:do_dump called with bad pos!')
```

Comment:
this should be an `error`, otherwise `getTileType` will throw anyway when you pass `nil` to it